nilearn/glm/first_level/first_level.py

```diff
@@ -1060,6 +1015,120 @@ def first_level_from_bids(dataset_path, task_label, space_label=None,
                                  (len(events), len(imgs)))
             confounds = [pd.read_csv(c, sep='\t', index_col=None)
                          for c in confounds]
+
             models_confounds.append(confounds)
 
     return models, models_run_imgs, models_events, models_confounds
+
+
+def _bids_validate_arguments(dataset_path: str,
```

Comment:
really unsure how to name those helper functions: prefixed them them `_bids_` for now to at least make it clearer what is the public function they relate to.

Reply:
Sounds OK.